news/2025/2025-08-15-project-loved-august-2025.md

```diff
@@ -0,0 +1,488 @@
+As the last note rang out and the scores were revealed, the margin was razor-thin.\
+A comeback carved out in the final breath of a song that thrives on tension, silence, and release.\
+At that moment, it wasn't just background music — it became the soul of the battle, a stage where precision met emotion, and where victory was won not with brute force, but with poise under pressure...
+
+Introducing rosso (Live at Kyoto Concert Hall 2019.10.19), from [Evening](https://osu.ppy.sh/users/2193881), which was the tiebreaker for the [osu!mania 7K World Cup 2024 Finals](/wiki/Tournaments/MWC/2024_7K).
+
+We think many people are already familiar with this map.\
+Various patterns suitable for tiebreakers and pattern designs that provide the best experience for the music, as Evening always does, further enhance the immersive experience of the music.\
+The impressive matches that took place between China and the Philippines for a spot in the Grand Final will also remain in everyone's memories.\
+We believe this is a great opportunity to relive those emotions or experience Evening's masterpiece once again.
+
+If you'd like to join this epic chapter in Mania history, please cast your vote to show your support!
+
+[![](/wiki/shared/news/2025-08-15-project-loved-august-2025/2115489.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Yorushika - Haru](https://osu.ppy.sh/beatmapsets/2115489#mania) by [Emida](https://osu.ppy.sh/users/11937683)\
+204 BPM, 4:09 | 7.79★\
+*description written by [Maxus](https://osu.ppy.sh/users/4335785)*
+
+We are absolutely delighted to present to you one of our most exclusive LN Pick of the Month maps. A spectacular chart created by none other than [Emida](https://osu.ppy.sh/users/11937683), a mapper whose name is already widely recognised within the mania community for their exceptional library of LN maps. For this special feature, we have carefully selected [Yorushika - Haru](https://osu.ppy.sh/beatmapsets/2115489#mania/4442387), a fan-favourite anime song from the franchise that gained massive popularity and even went viral at one point, making it a perfect choice for this showcase.
+
+The chart itself is a masterfully designed experience, brimming with a rich variety of LN skill sets that have been mapped with precision and cleanliness, all while incorporating the energetic and dynamic style of O2Jam. Whether you're looking for a map that challenges your technical abilities, offers creative patterning, or simply provides an incredibly fun and engaging playthrough, this chart has something to offer for everyone.
+
+So, what are you waiting for? If you enjoy the map and appreciate the effort that went into its creation, be sure to show your support by giving it a positive rating. It means a lot to the mapper and helps us bring you even more amazing picks in the future!
```

Comment:
> giving it a positive rating

that's not how loved works, this whole description reads like a generic somewhat-out-of-touch map promotion text

Reply:
https://github.com/ppy/osu-wiki/pull/13604/commits/ccf6c193e189b4208ed724e61a9775d001721793